iocore/net/UnixNetVConnection.cc

```diff
@@ -1129,9 +1130,14 @@ UnixNetVConnection::mainEvent(int event, Event *e)
     }
```

Comment:
This conditional never be true with this change? 
Because when this handler got `VC_EVENT_INACTIVITY_TIMEOUT`, `vc->inactivity_timeout_in && vc->next_inactivity_timeout_at <= now` is true.

Reply:
Yes, I suppose the condition was protecting from other arbitrary EVENT_IMMEDIATES coming through.